src/main/java/bobby/ui/Bobby.java

```diff
@@ -0,0 +1,366 @@
+package bobby.ui;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+
+/**
+ * Bobby is a chatbot that can manage a list of tasks, ToDos, Deadlines, and Events.
+ * It can mark tasks as done, unmark, delete and add tasks.
+ */
+public class Bobby {
+
+    /**
+     * Greeting message displayed when the bot starts
+     */
+    private static final String GREETING = "Hello! I'm Bobby\n"
+            + "What can I do for you?\n";
+
+    /**
+     * Exit msg when the bot terminates
+     */
+    private static final String EXIT = "Bye. Hope to see you again soon!";
+
+    /**
+     * Scanner to read user input
+     */
+    private static Scanner scan = new Scanner(System.in);
```

Comment:
Perhaps could rename scan to scanner for clarity and consistency with naming conventions?